test/solidity/Facets/GlacisFacet.t.sol

```diff
@@ -0,0 +1,284 @@
+                amount < 100_000 * 10 ** wormhole.decimals()
+        );
+        vm.startPrank(USER_SENDER);
+        bridgeData.minAmount = amount;
+        // approval
+        wormhole.approve(address(glacisFacet), bridgeData.minAmount);
+        QuoteSendInfo memory quoteSendInfo = IGlacisAirlift(address(airlift))
+            .quoteSend(
+                bridgeData.sendingAssetId,
+                bridgeData.minAmount,
+                bytes32(uint256(uint160(bridgeData.receiver))),
+                bridgeData.destinationChainId,
+                REFUND_WALLET,
+                payableAmount
+            );
+        addToMessageValue =
+            quoteSendInfo.gmpFee.nativeFee +
+            quoteSendInfo.airliftFeeInfo.airliftFee.nativeFee;
+        //prepare check for events
+        vm.expectEmit(true, true, true, true, address(glacisFacet));
+        emit LiFiTransferStarted(bridgeData);
+        initiateBridgeTxWithFacet(false);
+        vm.stopPrank();
+    }
+
+    function testBase_CanSwapAndBridgeNativeTokens() public override {
+        // facet does not support bridging of native assets
+    }
+
+    function setDefaultSwapDataSingleDAItoWORMHOLE() internal virtual {
+        delete swapData;
+        // Swap DAI -> WORMHOLE
+        address[] memory path = new address[](2);
+        path[0] = ADDRESS_DAI;
+        path[1] = ADDRESS_WORMHOLE_TOKEN;
+
+        uint256 amountOut = defaultWORMHOLEAmount;
+
+        // Calculate DAI amount
+        uint256[] memory amounts = uniswap.getAmountsIn(amountOut, path);
+        uint256 amountIn = amounts[0];
+
+        swapData.push(
+            LibSwap.SwapData({
+                callTo: address(uniswap),
+                approveTo: address(uniswap),
+                sendingAssetId: ADDRESS_DAI,
+                receivingAssetId: ADDRESS_WORMHOLE_TOKEN,
+                fromAmount: amountIn,
+                callData: abi.encodeWithSelector(
+                    uniswap.swapExactTokensForTokens.selector,
+                    amountIn,
+                    amountOut,
+                    path,
+                    _facetTestContractAddress,
+                    block.timestamp + 20 minutes
+                ),
+                requiresDeposit: true
+            })
+        );
+    }
+
+    function testBase_CanSwapAndBridgeTokens()
+        public
+        override
+        assertBalanceChange(ADDRESS_DAI, USER_RECEIVER, 0)
+        assertBalanceChange(ADDRESS_WORMHOLE_TOKEN, USER_SENDER, 0)
+        assertBalanceChange(ADDRESS_WORMHOLE_TOKEN, USER_RECEIVER, 0)
+    {
+        // add liquidity for dex pair
+        addLiquidity(
+            ADDRESS_DAI,
+            ADDRESS_WORMHOLE_TOKEN,
+            100_000 * 10 ** ERC20(ADDRESS_DAI).decimals(),
+            100_000 * 10 ** wormhole.decimals()
+        );
+
+        uint256 initialDAIBalance = dai.balanceOf(USER_SENDER);
+        vm.startPrank(USER_SENDER);
+        // prepare bridgeData
+        bridgeData.hasSourceSwaps = true;
```

Comment:
blank line after this and before // prepare bridgeData comment

Reply:
good point